BibliographyParser.py

```diff
@@ -0,0 +1,26 @@
+import re
+from typing import List, Dict
+
+class BibliographyParser:
+    def __init__(self):
+        self.globalPattern = re.compile(r"(\[([^\s]*)\])[^\S\n]{2,}([\s\S]*?(?=\n{2,}|\n[^\S\n]*\n|\[))")
+        self.postprocessPattern = re.compile(r"\s+")
+        self.symbolsToSkip = 0.65
```

Comment:
In future, we could try to dynamically find the cut of point. Few ideas
- If we parse the whole document, one section should have significantly more references than any other. That should be the bibliography.
- We could piggyback based on table of contents, when we implement parsing for that.

Those are ideas for future, if we see that we need to improve it.

Reply:
I was thinking about finding some keyword as "Bibliography" or "reference" but it should not be always true. Your suggestion seems reasonable.

Reply:
They might or might not work, it's just an idea. We can test it when we come to it. :)